redisinsight/api/provider-interfaces/models/agreements.ts

```diff
@@ -0,0 +1,14 @@
+export class Agreements {
```

Comment:
Q: why use named properties as opposed to a hashmap with string keys and boolean values ?